Makefile

```diff
@@ -75,23 +75,17 @@ prepare: clean
 	@echo "--- Prepare build directories ---"
 	@mkdir -p packaging/{sources,SRPMS}/
 
-# FIXME: incompatible with newer version of copr-cli
-_list_approved_builds:
-	@copr --config $(_COPR_CONFIG) get-package $(_COPR_REPO) \
-		--name $(__PKGNAME) --with-all-builds \
-		| grep -E '"(built_packages|id|state|pkg_version)"' | grep -B3 "succeeded" \
-		| sed 's/"state": "succeeded",/----------------------/' \
-		| grep -A1 -B2 '"pkg_version".*-[1-9]'
-
 source: prepare
 	@echo "--- Create source tarball ---"
 	@echo git archive --prefix "$(PKGNAME)-$(VERSION)/" -o "packaging/sources/$(PKGNAME)-$(VERSION).tar.gz" HEAD
 	@git archive --prefix "$(PKGNAME)-$(VERSION)/" -o "packaging/sources/$(PKGNAME)-$(VERSION).tar.gz" HEAD
 	@echo "--- PREPARE DEPS PKGS ---"
 	mkdir -p packaging/tmp/
 	@__TIMESTAMP=$(TIMESTAMP) $(MAKE) _copr_build_deps_subpkg
-	@copr --config $(_COPR_CONFIG) download-build -d packaging/tmp \
-		`COPR_REPO=$(_COPR_REPO_TMP) COPR_PACKAGE=$(DEPS_PKGNAME) ./utils/get_latest_copr_build --id`
+	@PKG_RELEASE=$(RELEASE) _COPR_CONFIG=$(_COPR_CONFIG) \
+		COPR_REPO=$(_COPR_REPO_TMP) COPR_PACKAGE=$(DEPS_PKGNAME) \
+		./utils/get_latest_copr_build --id > packaging/tmp/deps_build_id
+	@copr --config $(_COPR_CONFIG) download-build -d packaging/tmp `cat packaging/tmp/deps_build_id`
 	@mv `find packaging/tmp/ | grep "rpm$$" | grep -v "src"` packaging/tmp
 	@tar -czf packaging/sources/deps-pkgs.tar.gz -C packaging/tmp/ `ls packaging/tmp | grep -o "[^/]*rpm$$"`
```

Comment:
If subshells aren't working, why ware these next two lines working ;-)

Reply:
Or your cat

Reply:
I wrote, nested subshell :) Example:
```
 $ echo  `echo something`
 $ # vs
 $ echo `echo `echo somethin``
```
Try to expand `$(RELEASE)`